admin/data-collection.rst

```diff
@@ -152,6 +152,78 @@ example, you cannot use operations like ``less than`` or ``greater than`` on
 strings. If you select metric from the list using the :guilabel:`Select` button,
 correct data type will be set automatically.
 
+
+Units
+~~~~~
+
+For user convenience collected DCI values can have the following units assigned:
+

```

Comment:
This list displays predefined units, but it's possible to enter any unit you like.